src/Catalyst.Core.Modules.Dfs.Tests/DagNodeTest.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Catalyst.Abstractions.Dfs;
+using Google.Protobuf;
+using MultiFormats;
+using Xunit;
+
+namespace Catalyst.Core.Modules.Dfs.Tests
+{
+    public class DagNodeTest
+    {
+        [Fact]
+        public void EmptyDAG()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'EmptyDAG' to match pascal case naming rules, consider using 'EmptyDag'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)